config/fixes/convention-eccodes.yaml

```diff
@@ -416,6 +427,7 @@ convention_name:
           - mtnlwrf
           - ttr
           - TTR # AQUA ERA5
+          # - rlut # cmor --> sign wrong
```

Comment:
as above, what we do in this case?

Reply:
Either we live with it (which means that the convention does not cover these variables and a specific fix needs to be specified), or we allow also the 'derived' key to have a list of formulas (and they could then be included in the convention file, or we develop the possibility to specify a - sign also for 'sources' (basically making it a bit similar to 'derived')